components/background_hang_monitor/sampler_linux.rs

```diff
@@ -18,7 +18,7 @@ use crate::sampler::{NativeStack, Sampler};
 // Hack to workaround broken libunwind pkg-config contents for <1.1-3ubuntu.1.
 // https://bugs.launchpad.net/ubuntu/+source/libunwind/+bug/1336912
 #[link(name = "lzma")]
-extern "C" {}
+unsafe extern "C" {}
```

Comment:
The link says this bug was fixed in 2014, so I wonder if we can just remove this block. Might be worth to follow-up as an issue.